pkg/apis/crd.nsx.vmware.com/v1alpha1/ipblockinfo_types.go

```diff
@@ -0,0 +1,42 @@
+/* Copyright © 2024 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+//+kubebuilder:object:root=true
+//+kubebuilder:resource:scope="Cluster"
+
+// IPBlockInfo is the Schema for the ipblockinfo API
+type IPBlockInfo struct {
```

Comment:
IPBlocksInfo as it includes multiple IPBlocks?

Reply:
Updated, thanks!